sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala

```diff
@@ -177,11 +179,11 @@ class SourceProgress protected[sql](
     }
 
     ("description" -> JString(description)) ~
-      ("startOffset" -> tryParse(startOffset)) ~
-      ("endOffset" -> tryParse(endOffset)) ~
-      ("numInputRows" -> JInt(numInputRows)) ~
-      ("inputRowsPerSecond" -> safeDoubleToJValue(inputRowsPerSecond)) ~
-      ("processedRowsPerSecond" -> safeDoubleToJValue(processedRowsPerSecond))
```

Comment:
nit, the original style seems to be more frequent and looks correct, doesn't it?

Reply:
I though it'd been the opposite - see https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala#L53-L57 and https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/streaming/progress.scala#L128-L140.

I've made the section to match the style of the others in the source file.

Reply:
Yep. I guess that those are the less frequent style.